slave-maven/test/run

```diff
@@ -5,8 +5,13 @@
 # IMAGE_NAME specifies the name of the candidate image used for testing.
 # The image has to be available before this script is executed.
 #
+# TODO: When the rhel image get gradle remove the if
+#
 
 docker run ${IMAGE_NAME} oc
 docker run --entrypoint="/bin/bash" ${IMAGE_NAME} "-c" "mvn --version"
+if [[ ${IMAGE_NAME} != *"rhel"* ]]; then
+    docker run --entrypoint="/bin/bash" ${IMAGE_NAME} "-c" "gradle --version"
```

Comment:
@bparees no problem have been trying to find why the test didn't work. Now it should work fine